fix: 마이페이지 화면 수정 및 이미지 업로드 리팩토링

gffd94 opened the pull request:
## 관련 이슈

close #61 

## 개요

1. 자기 자신인 경우 owner : 프로필 편집 버튼 활성화 아닌 경우 팔로우 버튼
2. 이미지 업로드 방식 String -> MultipartFile로 변경
3. 이미지 변경과 동시에 마이페이지에 반영(MemberUpdateResponse)
4. 마이페이지에서 저장된 정보가 다른화면으로 이동시 초기화되는 현상 해결

gffd94 committed:
:sparkles: fix: 마이페이지 위치수정[FE]
:recycle: refactor: 프로필 변경방식에 따른 리팩토링[BE]

gffd94 committed:
:sparkles: fix: 마이페이지 위치수정[FE]
:recycle: refactor: 프로필 변경방식에 따른 리팩토링[BE]

Commit:
chore: Java 스타일 수정

Commit:
chore: Java 스타일 수정

Jeongmin39 committed:
feat: 좋아요 API 구현 및 연동 (#68)

* feat: 좋아요 누르기 및 취소 api 구현

* feat: 좋아요 누르기 및 취소 api 연동

* chore: Java 스타일 수정

* refactor: 로그인 상태에서만 좋아요 누를 수 있도록 수정 및 알림 추가

---------

Co-authored-by: github-actions <>

dlsrks1021 commented on NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/domain/Member.java:
```diff
@@ -35,8 +35,18 @@ public class Member extends BaseEntity {
 
-    public void updateProfile(String profileImage) {
-        this.profileImage = profileImage;
+    //    public void updateProfile(String profileImage, String nickname) {
```
[L4-변경제안]
아래 메서드가 구현되어 주석은 제거해도 괜찮을 것 같습니다.

gffd94 replied:
주석 제거했습니다.

dlsrks1021 commented on NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/domain/Member.java:
```diff
@@ -35,8 +35,18 @@ public class Member extends BaseEntity {
+    //    }
+
+    public void updateProfile(String nickname, String profileImageUrl) {
+        if (nickname != null && !nickname.isBlank()) {
```
[L5-참고의견]
Apache Commons의 StringUtils.isBlank()를 사용해 볼 수 있을 것 같습니다.
https://hyeri0903.tistory.com/235

gffd94 replied:
수정했습니다!

dlsrks1021 commented on NBE_5_7_2_02TEAM/uploads/member/1/41d55ec2-efe5-4b48-ab4f-ba3e96dbfb1b_스크린샷 2025-05-16 105716.png:
[L2-변경협의]
uploads의 샘플 이미지들은 git 변경사항에서는 삭제되는 것이 좋을 것 같습니다.

gffd94 replied:
이거 ... 어떻게 제거하나요?

dlsrks1021 replied:
`git rm --cached -r uploads/`
이후 커밋 작성하시면 될 것 같습니다.

gffd94 replied:
일단 잘 제거된거 같네요 ㅎㅎ 감사합니다

gffd94 committed:
fix:  PR 리뷰수정

# Conflicts:
#	NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/domain/Member.java

gffd94 committed:
fix:  PR 리뷰수정

# Conflicts:
#	NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/domain/Member.java

Commit:
chore: Java 스타일 수정

gffd94 committed:
:sparkles: fix: 마이페이지 위치수정[FE]
:recycle: refactor: 프로필 변경방식에 따른 리팩토링[BE]

gffd94 committed:
PR 리뷰수정

Commit:
chore: Java 스타일 수정
